model/gen-ai/registry.yaml

```diff
@@ -5,7 +5,7 @@ groups:
     brief: >
       This document defines the attributes used to describe telemetry in the context of Generative Artificial Intelligence (GenAI) Models requests and responses.
     attributes:
-      - id: gen_ai.system
+      - id: gen_ai.system.name
```

Comment:
This is creates some confusion with `gen_ai.system.message` https://github.com/open-telemetry/semantic-conventions/blob/beb66215d37557d8bfc9303abd4ce3e2ef47da01/model/gen-ai/events.yaml#L10-L17